src/pages/RadicalCollaboration/FAQCardDesktop.js

```diff
@@ -35,8 +35,8 @@ const FAQCardDesktop = (props) => {
                 </Typography>
 
                 <Typography className={classes.faqLinkText}>
-                  View our  <Link to="/faq">FAQ</Link>
-                   to find answers or <a className={classes.inLineLinkText} href='/contact'>contact us.</a>
+                  View our   <Link to="/faq"> FAQ  </Link>
+                  <span> to find answers or <a className={classes.inLineLinkText} href='/contact'>contact us.</a> </span>
```

Comment:
is "/contact" an internal or external link?  If internal, then can it be replaced with a Link component?

Reply:
I think you are seeing the wrong PR , The code I updated on "Refactored_Collaborate_With_Us" title which has the recent code